src/testFixtures/workspaceFolder/go1-image-sam-app/hello-world/Dockerfile

```diff
@@ -0,0 +1,13 @@
+FROM golang:1.14 as build-image
+
+WORKDIR /go/src
+COPY go.mod main.go ./
+
+RUN go build -o ../bin
```

Comment:
We can stick `GOPROXY=DIRECT` in here instead, see JB: https://github.com/aws/aws-toolkit-jetbrains/blob/master/testdata/samProjects/image/go1.x/Dockerfile#L3

Reply:
Yeah that makes more sense.